source/numerics.tex

```diff
@@ -8228,7 +8228,7 @@
 refers.
 \end{itemdescr}
 
-\rSec3[gslice.array.comp.assign]{\tcode{gslice_array}}
+\rSec3[gslice.array.comp.assign]{\tcode{gslice_array} computed assignment}
```

Comment:
@zygoloid I'm not sure if it is correct. "Computed assignments" occurs in many other clauses, i.e. [valarray.syn]/5, [template.valarray.overview], [valarray.cassign], [slice.arr.comp.assign], [mask.array.comp.assign] and [indirect.array.comp.assign]. Is this editorial? Also not sure difference between "cassign" and "comp.assign" is intended.